K8s/helm/README.md

```diff
@@ -95,3 +95,6 @@ $ helm install robot-shop --set openshift=true helm
 | psp.enabled      | false | boolean | Enable Pod Security Policy for clusters with a PSP Admission controller |
 | redis.storageClassName | standard | string | Storage class to use with Redis's StatefulSet. The default for EKS is gp2. |
 | ocCreateRoute    | false | boolean | If you are running on OpenShift and need a Route to the web service, set this to `true` |
+| affinity    | {} | object | Affinity for pod assignment on nodes with matching labels (Refer [here](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity)) |
+| nodeSelector    | {} | object | Node labels for pod assignment (Refer [here](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector)) |
+| tolerations    | [] | list | Tolerations for pod assignment on nodes with matching taints (Refer [here](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/)) |
```

Comment:
I suggest to add another section called "Deployment Examples", in which you explain with one or two scenarios how to distribute the workloads in deployments examples

Reply:
@teresanovielloatinstana 
I have updated the docs here: https://github.com/instana/robot-shop/blob/6a2fa13cdd1b819c68742a68bee307491de34bd6/K8s/helm/README.md
Let me know if that will suffice. Thanks for the suggestion. :)